doc/source/whatsnew/v0.17.0.txt

```diff
@@ -54,6 +54,7 @@ Other API Changes
 - Enable serialization of lists and dicts to strings in ExcelWriter (:issue:`8188`)
 - Allow passing `kwargs` to the interpolation methods (:issue:`10378`).
 - Serialize metadata properties of subclasses of pandas objects (:issue:`10553`).
+- Clarifying ``display.precision`` option to refer to decimal places and correspondingly changing default value to ``6`` (:issue:`10451`)
```

Comment:
can you add an example of the before (use a code-block) and the after (use ipython block). Just to make it obvious what is changing.